simple_trajectories/src/main.cpp

```diff
@@ -73,10 +76,11 @@ void moveArmTo(
   armSpace->convertStateToPositions(state, positions);
   ROS_INFO_STREAM(positions.transpose());
 
-  auto smoothTrajectory
-      = robot.smoothPath(armSkeleton, trajectory.get(), satisfied);
+  auto smoothTrajectory = robot.postProcessPath<ParabolicSmoother>(
+      trajectory.get(), satisfied, ParabolicSmoother::Params());
   aikido::trajectory::TrajectoryPtr timedTrajectory
-      = std::move(robot.retimePath(armSkeleton, smoothTrajectory.get()));
+      = std::move(robot.postProcessPath<KunzRetimer>(
+          smoothTrajectory.get(), satisfied, KunzRetimer::Params()));
```

Comment:
Should this be changed to use the ADA-specific `Kunz` params struct introduced in `libada`?